src/Form/Question.php

```diff
@@ -44,4 +45,20 @@ class Question extends CommonDBChild
 {
     public static $itemtype = Section::class;
     public static $items_id = 'forms_sections_id';
+
+    /**
+     * Get type object for the current object.
+     *
+     * @return QuestionTypeInterface|null
+     */
+    public function getQuestionType(): ?QuestionTypeInterface
+    {
+        $type = $this->fields['type'] ?? "";
+
+        if (!is_a($type, QuestionTypeInterface::class, true)) {
+            return null;
+        }
+
+        return new $type();
```

Comment:
`QuestionTypeInterface` should define a constructor without any parameters.

``public function __construct();``